commands.sh

```diff
@@ -89,6 +89,10 @@ alias tdrs-fix-django-populate='tdrs-backend-exec && sed -i "s/raise Runtime..po
 # A recurring pattern I was doing to get a fresh setup
 alias tdrs-fresh-start='tdrs-stop && docker system prune --volumes && tdrs-start'
 
+# Deploy current branch to the given environment
+# See comments in main function for to set up Circle CI token
+alias tdrs-deploy='tdrs-run-deploy'
```

Comment:
minor documentation change for team members that spin up apps locally cc: @ttran-hub